README.md

```diff
@@ -66,24 +68,38 @@ Or you can add full namespace and collection name in the `collections` element:
         key: example_flag_creation
 ```
 
-## Resource Supported
-
-- launchdarkly_feature_flag - Create, update, delete a LaunchDarkly Feature Flag
-- launchdarkly_feature_flag_environment - Configure an Environment specific LaunchDarkly Feature Flag
-- launchdarkly_feature_flag_sync - Sync flag settings across environments
-- launchdarkly_custom_role - Create, update, delete a LaunchDarkly Custom Role
-- launchdarkly_environment - Create, update, delete a LaunchDarkly Project Environment
-- launchdarkly_project - Create, update, delete a LaunchDarkly Project
-- launchdarkly_project_copy - Copy a LaunchDarkly Project
-- launchdarkly_test_generator - Generate a LaunchDarkly compatible SDK test file
-- launchdarkly_user_segment - Create, update, delete a LaunchDarkly User Segment
-- launchdarkly_user_segment_sync - Copy a LaunchDarkly User Segment across Environments
-- launchdarkly_variation_info - Return a value from a feature flag evaluation
-- launchdarkly_webhook - Create, update, delete a LaunchDarkly Webhook
+## Resources supported
+
+The LaunchDarkly Ansible collection supports the following resources:
+
+- `launchdarkly_custom_role`: Manage custom roles
```

Comment:
Reordered alphabetically, to match how these resources appear in the generated documentation. Added resources that are available in the generated documentation but missing here.